docs/op-stack/src/docs/build/getting-started.md

```diff
@@ -390,9 +390,7 @@ Head over to the `op-node` package and start the `op-node` using the following c
 	--rollup.config=./rollup.json \
 	--rpc.addr=0.0.0.0 \
 	--rpc.port=8547 \
-	--p2p.listen.ip=0.0.0.0 \
-	--p2p.listen.tcp=9003 \
-	--p2p.listen.udp=9003 \
+    --p2p.disable \
```

Comment:
Nit: this is offset

Reply:
Hmmm would be ideal if you can just copy/paste one of the tabs from above to resolve this